universe/index.js

```diff
@@ -0,0 +1,49 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of vikram.hello
+//
+// Copyright 2021 undefined <undefined>
+//
+// See LICENSE for details
+"use strict";
+const Tp = require("thingpedia");
+const querystring = require("querystring");
+const assert = require('assert');
+
+const tmdbAccess = "https://api.themoviedb.org/3/";
+const movieSearch = "search/movie?api_key=654083fa9e049ea234e4ae94d3e65774&language=en-US&query=";
```

Comment:
Don't hard code the API key here, because it becomes visible to anyone on the internet and can be stolen. Instead, create a file called `secrets.json` containing:
```
{"api_key": "12345678"}
```

Then, in the manifest, add:
```
import config from @org.thingpedia.config.none(api_key=$?);
```
inside the class.

Finally, you can read out the API key inside the device class with `this.constructor.metadata.auth.api_key`.